
(checker.py:6623): dbind-WARNING **: 02:22:38.931: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-yrsWVO5zZb: Connection refused
